src/psyclone/tests/gocean1p0_opencl_test.py

```diff
@@ -433,15 +569,54 @@ def test_set_arg_const_scalar():
 @pytest.mark.usefixtures("kernel_outputdir")
 def test_opencl_kernel_code_generation():
     ''' Tests that gen_ocl method of the GOcean Kernel Schedule generates
-    the expected OpenCL code.
+    the expected OpenCL code. Note this test doesn't prepare the kernel to
+    conform to OpenCL GOcean expected interface yet.
```

Comment:
s/yet./: see test_opencl_prepared_kernel_code_generation for that./

Reply:
Done